index.html

```diff
@@ -206,7 +206,7 @@ <h2>CONTACT US</h2>
         </a>
     </div>
      <div class="email-link">
-         <a href="#">kinyeramo@gmail.com</a>
+         <a href="#" class="email-link">kinyeramo@gmail.com</a>
      </div>
```

Comment:
Its good to see you have implement the contact form. Kindly remove the code in your html file from line 176 -210 for it is not part of the project requirements for this week. The contact form will be a project you will do in the coming weeks. Please lets stick to project requirements.